lib/vagrant-service-manager/command.rb

```diff
@@ -266,21 +266,21 @@ def print_vagrant_box_version(script_readable = false)
       end
 
       def restart_service(service)
-
-        if SCCLI_SERVICES.include? service
-          # TODO : Handle the case where user wants to pass extra arguments to OpenShift service
-          command = "sudo sccli #{service}"
-        else
-          command = "sudo systemctl restart #{service}"
-        end
-
-        with_target_vms(nil, { single_target: true}) do |machine|
-          machine.communicate.execute(command) do |type, error|
-            if type == :stderr
-              @env.ui.error(error)
-              exit 126
-            end
+        command = if SCCLI_SERVICES.include? service
+                    # TODO : Handle the case where user wants to pass extra
+                    # arguments to OpenShift service
+                    "sccli #{service}"
+                  else
+                    "systemctl restart #{service}"
+                  end
+
+        with_target_vms(nil, single_target: true) do |machine|
+          exit_code = machine.communicate.sudo(command)
+          unless exit_code.zero?
+            @env.ui.error "#{command} failed"
```

Comment:
> require 'open3'
stdout, stdeerr, status = Open3.capture3("ls")
puts stdout

This code looks like hack over tradition Vagrant way of streams which I mentioned [in this comment](https://github.com/projectatomic/vagrant-service-manager/pull/167#discussion_r59328368) and I wouldn't like to vote for it.